### PR TITLE
Add contribution guidelines

### PR DESCRIPTION
**Pull request by oleiade**
This PR adds a very rough set of contribution guidelines/instructions.

@immavalls please feel free to poke holes in it, point out all the stuff that's missing, questions you have that might not be answered by it, improvements ideas, everything 👍🏻 🤝 🙇🏻

### CODE REVIEW

File: CONTRIBUTING.md
Comment (immavalls) [inline CONTRIBUTING.md:47]: - Should we also bump `package-lock.json`? 
- Is this done directly on the main branch once the PR is approved, as it might include more PRs a new version?

File: CONTRIBUTING.md
Comment (immavalls) [inline CONTRIBUTING.md:49]: Is this referring to creating a tag for the new version under https://github.com/grafana/k6-jslib-aws/tags?

File: CONTRIBUTING.md
Comment (oleiade) [reply CONTRIBUTING.md:50]: ✅ I have added an instruction step about updating the `package-lock.json`
✅ I've made it more explicit that this should happen in a PR (which maintainers should feel free to directly merge without reviews)

🙇🏻

File: CONTRIBUTING.md
Comment (oleiade) [reply CONTRIBUTING.md]: I've made it more explicit that we follow the semantic versioning convention, and that tags and releases are expected to follow that convention 👍🏻

File: CONTRIBUTING.md
Comment (oleiade) [reply CONTRIBUTING.md]: Exactly 👍🏻 I've tried to make this more explicit indeed 🙇🏻

**Review by immavalls — APPROVED** [review]
Looks good to me, I was able to follow the process, thanks!

**Pull request merged**
